independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/CreateProject.java

```diff
@@ -176,7 +183,15 @@ public QuarkusCommandOutcome execute() throws QuarkusCommandException {
         } else {
             setValue(JAVA_TARGET, "11");
         }
-
+        if (containsSpringWeb(extensions)) {
+            setValue(IS_SPRING, true);
+            if (containsRESTEasy(extensions)) {
+                values.remove(CLASS_NAME);
+                values.remove(PACKAGE_NAME);
+                values.remove(RESOURCE_PATH);
```

Comment:
I assume that's necessary but it's not clear why. In case of Spring these props aren't relevant? Are they still relevant for RESTEasy?

Reply:
@aloubyansky there is another `if (containsSpringWeb(extensions)) {` just before, so if we have `resteasy` and `spring` we don't use those because it would means that the spring resources and the resteasy one will have the same path and class name, we don't want that :)

Reply:
if you have just `resteasy` or `spring` then there is no problem :)

Reply:
Are these CLASS_NAME, etc used in case of Spring web at all? How is the class name passed for spring?

Reply:
yes: https://github.com/ia3andy/quarkus/blob/fix-spring-web-codestart/independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/codestarts/QuarkusCodestartData.java#L78-L80